pkg/engine/engine.go

```diff
@@ -450,14 +449,31 @@ func (e *Engine) Logs(ctx context.Context, id string, follow bool, cancel bool,
 		}
 	}
 
-	file, err := os.Open(path)
+	stop := make(chan struct{})
+	file, err := newTailReader(path, stop)
```

Comment:
@hacdias @raulk if you have better ideas on how to mimic `tail -f` behaviour, I'd be happy to refactor this.

The idea here is that we continuously read the file (which includes logs for a given task), and we sleep a bit if we encounter `io.EOF`.

`engine.Logs` is blocking, and `e.signals[id]` governs when we return from this function - this is used for the `--wait` flag, as well for the `testground logs --follow` command.

Reply:
I think we should get rid of the tailing entirely. The way I see it:
1. we have run events in the sync service to update the user view.
2. we should consume those run events live, and update the UI accordingly. IMO that's what the new "tail" should be.
3. results aren't really actionable/useful until the run has finished, so those should be downloaded as outputs.